validator/client/runner.go

```diff
@@ -62,18 +62,14 @@ func run(ctx context.Context, v Validator) {
 			log.Fatalf("Slasher is not ready: %v", err)
 		}
 	}
-	if featureconfig.Get().WaitForSynced {
-		if err := v.WaitForSynced(ctx); err != nil {
```

Comment:
Are you going to delete this method too?

Reply:
This feature looks like it wasn't launched yet, but you are removing it?

Reply:
cc: @0xKiwi

Reply:
Similar to `StateGenSigVerify`, this looks to be a feature that we should never release on beta/mainnet. This skips WaitForChainStart and is only utilized during test (e.g. e2e)

Reply:
Chatted with @0xKiwi this feature is not done. We'll leave it in for now